sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -335,12 +335,13 @@ object Radar {
     private lateinit var context: Context
     private lateinit var handler: Handler
     private var receiver: RadarReceiver? = null
-    internal lateinit var logger: RadarLogger
+    internal val logger = RadarLogger()
```

Comment:
what's the thinking behind making this variable non-lazy?

Reply:
ah, got it, I misread this. still trying to see what the value of this over consistency with other variables is

Reply:
ah, it's a static global logger. I suppose we could use the singleton pattern of checking if it's null in the constructor but this is fine